src/custom/constants/subsidy.ts

```diff
@@ -0,0 +1,10 @@
+export const COW_SUBSIDY = [
+  ['(v)COW balance', 'Fee discount'],
```

Comment:
I find it strange to add the headers here, why is this?  Can this be part of the content together with the other content instead fo here mixed with the data?